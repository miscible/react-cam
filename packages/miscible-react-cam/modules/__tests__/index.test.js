import React from "react";
import renderer from "react-test-renderer";
import Camera from "../index";

test("Link changes the class when hovered", () => {
  const component = renderer.create(<Camera />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});
