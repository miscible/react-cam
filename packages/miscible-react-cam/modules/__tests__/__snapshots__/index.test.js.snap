// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<video
  autoPlay={true}
  height={320}
  src={null}
  style={
    Object {
      "WebkitTransition": "scaleX(-1)",
      "transform": "scaleX(-1)",
    }
  }
  width={640}
/>
`;
